packages/vscode-extension/src/common/License.ts

```diff
@@ -0,0 +1,27 @@
+// #region License Helpers
+
+export function hasAccessToProFeatures(licenseStatus: LicenseStatus) {
+  return (
+    licenseStatus === LicenseStatus.Pro ||
+    licenseStatus === LicenseStatus.Team ||
+    licenseStatus === LicenseStatus.Enterprise
+  );
+}
```

Comment:
_I'd prefer_ if this function instead took:

```
export function hasAccessToFeature(licenseStatus: LicenseStatus, feature: Feature) {
```
for some 
```
enum Feature {
  DeviceRotation,
  TabletEmulators,
  /* etc... */
}
```
and then have a map from `LicenseStatus` to `Set<Feature>` that you use to check feature support.

- it makes changing which features are in which tier easier, since you don't have to look for each guard for that feature, you just change one entry in the map.
- it would allow us to easily change where we store the information. I'd imagine at some point we may want to move the feature sets to the customer portal, so that we can make changes without pushing Radon updates. It would also make it possible to allow enterprise admins to block features in their orgs.